domain_1.2.14.go

```diff
@@ -70,3 +71,82 @@ func (d *VirDomain) ListAllInterfaceAddresses(src uint) ([]VirDomainInterface, e
 	C.free(unsafe.Pointer(cList))
 	return ifaces, nil
 }
+
+func (dest *VirTypedParameters) loadToCPtr() (params C.virTypedParameterPtr, nParams C.int, err error) {
+
+	var maxParams C.int = 0
+	params = nil
+
+	for _, param := range *dest {
+
+		switch param.Name {
+
+		case VIR_DOMAIN_BLOCK_COPY_GRANULARITY:
+			value, ok := param.Value.(uint)
+			if !ok {
+				err = errors.New("Mismatched parameter value type")
+
```

Comment:
We don't have this empty line between the code and return statements. Could you remove all newlines before the return statements?